Dockerfile

```
@@ -18,37 +18,35 @@ COPY cmd/    cmd/
 
 # Build
 RUN CGO_ENABLED=0 GOOS=linux \
-		go build \
-            -mod readonly \
-			-ldflags "$GO_LDFLAGS" -tags="$GO_TAGS" -a \
-			-o elastic-operator github.com/elastic/cloud-on-k8s/cmd
+      go build \
+      -mod readonly \
+      -ldflags "$GO_LDFLAGS" -tags="$GO_TAGS" -a \
+      -o elastic-operator github.com/elastic/cloud-on-k8s/cmd
 
 # Copy the operator binary into a lighter image
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
+FROM gcr.io/distroless/static:nonroot
 
 ARG VERSION
 
-# Add required ECK labels and override labels from base image
-LABEL name="Elastic Cloud on Kubernetes" \
+# Add common ECK labels and OCI annotations to image
+LABEL io.k8s.description="Elastic Cloud on Kubernetes automates the deployment, provisioning, management, and orchestration of Elasticsearch, Kibana, APM Server, Beats, and Enterprise Search on Kubernetes" \
       io.k8s.display-name="Elastic Cloud on Kubernetes" \
-      maintainer="eck@elastic.co" \
-      vendor="Elastic" \
-      version="$VERSION" \
-      url="https://www.elastic.co/guide/en/cloud-on-k8s/" \
-      summary="Run Elasticsearch, Kibana, APM Server, Enterprise Search, and Beats on Kubernetes and OpenShift" \
-      description="Elastic Cloud on Kubernetes automates the deployment, provisioning, management, and orchestration of Elasticsearch, Kibana, APM Server, Beats, and Enterprise Search on Kubernetes" \
-      io.k8s.description="Elastic Cloud on Kubernetes automates the deployment, provisioning, management, and orchestration of Elasticsearch, Kibana, APM Server, Beats, and Enterprise Search on Kubernetes"
-
-# Update the base image packages to the latest versions
-RUN microdnf update --setopt=tsflags=nodocs && microdnf clean all
+      org.opencontainers.image.authors="eck@elastic.co" \
+      org.opencontainers.image.base.name="gcr.io/distroless/static:nonroot" \
+      org.opencontainers.image.description="Elastic Cloud on Kubernetes automates the deployment, provisioning, management, and orchestration of Elasticsearch, Kibana, APM Server, Beats, and Enterprise Search on Kubernetes" \
+      org.opencontainers.image.documentation="https://www.elastic.co/guide/en/cloud-on-k8s/" \
+      org.opencontainers.image.licenses="Elastic-2.0" \
```

```suggestion
      org.opencontainers.image.licenses="Elastic-License-2.0" \
```
To be consistent with how Elasticsearch stylises this.